action.yml

```diff
@@ -1,76 +1,40 @@
 # action.yml
-name: 'lazy-action-dotnet'
-description: 'Github action to perform build, test , scan and generate image.'
+name: "lazy-action-dotnet"
```

Comment:
Should name be a single word? If not, then can we call it as `Lazy Action Dotnet`

Reply:
Updated.